README.md

```diff
@@ -211,6 +211,32 @@ Example usage:
   lines so you'll need to be careful to reconstruct this
 - `rails debug:saml['tmp/your_saml_to_debug.txt']`
 
+## Data Loading
+
+There are three types of data that get loaded into this system:
+
+### Database Seeds
+
+These should only be loaded when the application database is initially set up (e.g. for new PR/development deploys or if the staging database needs to be destroyed and recreated). These seeds contain default values for certain tables such as copyrights, licenses, hold sources, and degree types.
+
+The above seed data is loaded automatically during PR builds from Github. During local development it can be loaded during first deployment by running `rake db:seed`.
+
+Additionally, degrees and departments can be manually seeded from a CSV file if desired by running `rake db:seed_degrees <csv_file_url>` and `rake db:seed_departments <csv_file_url>`, respectively. See Jira project documentation for link to a Google doc with the initial list of departments and degrees that were loaded into the production database (not maintained).
+
+Seed data is not maintained to match the production database values, which can be changed by admin users as needed. The production database *shouldn't* ever need to be reseeded.
+
+### Test Data
```

Comment:
I think we may want to rephrase this heading. We have lots of data in our testing env already that we build up via fixtures or other means. I think this is referring to something like QA Data or something? (Words are not my friend right now)